.golangci.yml

```diff
@@ -44,13 +46,22 @@ linters:
     - exportloopref
     - gocritic
     - gofmt
-    - golint
     - misspell
     - prealloc
+    - revive
     - unconvert
     - whitespace
-    - maligned
 
     # To consider enabling in future (once we fix errors etc):
     # - errcheck
     # - scopelint
+
+linters-settings:
+  govet:
+    enable-all: true
+    disable:
+      - deepequalerrors # Only seen this once and it's a false positive
+      - unusedwrite # Similarly
+      - shadow # Probably not useful enough to clean everything up
+      # TODO(peterebden): Should clean up these warnings & enable (although they are pretty finicky)
+      - fieldalignment
```

Comment:
This is `malalign` right?

Reply:
Yes, it also has an extra check about GC pointer size which is part of the same checker (but quite a bit more subtle)